app/test/setup-test-framework.ts

```diff
@@ -3,3 +3,6 @@ jest.setTimeout(10000)
 
 import 'jest-extended'
 import 'jest-localstorage-mock'
+
+delete process.env['LOCAL_GIT_DIRECTORY']
```

Comment:
Why is this needed? I figured tests must be breaking.. but just curious.

Reply:
Yeah, see https://github.com/desktop/desktop/pull/14335/commits/2e80e8bc952366d654ead095381d6b0a15f4c6d0

> I had launched VS Code via Desktop and it had inherited these causing tests to use the prod version dugite

Reply:
So this change isn't necessary on CI but it's gonna bite me again one of these days so I'd rather fix it now